bff-node/src/util/InitializaitonUtils.ts

```diff
@@ -3,11 +3,13 @@ import '../config/env';
 import { DefaultAzureCredential } from '@azure/identity';
 import { AppConfigurationClient } from '@azure/app-configuration';
 import * as appInsights from 'applicationinsights';
-import { bffVersion } from '..';
+import { bffVersion, isLocal } from '..';
+import { resolve } from 'path';
 
 const debug = true;
 
 export const initAppInsights = async () => {
```

Comment:
I believe that it can quickly become challenging to maintain this file because it will constantly grow. It only contains a bunch of methods related to the initialization of various dependencies. What do you think about splitting this into multiple files, one file per 'domain'? For example, an ApplicationInsightsImpl class that handles Application Insights.

This can also make it easier for future developers to search for Application Insights in the future, as they can probably search the file to find the implementation